Tests/StreamChatUITests/SnapshotVariation.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatUI
+import XCTest
+
+enum SnapshotVariation: String, Hashable, CaseIterable {
```

Comment:
IMHO it's weird to joint Font Size and User Interface Style.
I would expect SnapshotVariation be at least pair:
```
static let default = (.large, .light)
static let bigAndDark = (.extraExtraExtraLarge, .dark)
...
```